external/mrtrix/mrtrix_track.m

```diff
@@ -1,8 +1,8 @@
-function [status, results, fg, pathstr] = mrtrix_track(files, roi, mask, mode, nSeeds, bkgrnd, verbose, clobber)
+function [status, results, fg, pathstr] = mrtrix_track(files, roi, mask, mode, nSeeds, curvature, cutoff, bkgrnd, verbose, clobber)
 %
 % function [status, results, fg, pathstr] = mrtrix_track(files, roi, mask, mode, nSeeds, bkgrnd, verbose)
 %
-% Provided a csd estimate, generate estimates of the fibers starting in roi 
+% Provided a csd estimate, generate estimates of the starting in roi 
```

Comment:
"Streamlines" sounds good to me. The words "fiber" and "fibers" appear in
many places in the vistasoft code-base. AFQ even stands for automated
*fiber* quantification! The word "streamlines" is more neutral somehow, and
therefore feels more "accurate", but what if you generated them in a
process that isn't an analogy to flow fields? Ultimately, these things are
sets in R3 that may or may not represent bundles, fascicles (a word I have
used myself!), fibers, axons, nerves, tracts, pathways, and any other
number of...  FWIW, we rather consistently refer to these as "streamlines"
in Dipy, and I know that many in the field like that word, if only because
it feels neutral, and allows these things to be models of different things,
depending on the way they were generated (data and algorithms).

On Mon, Dec 21, 2015 at 1:38 AM, Hiromasa Takemura <notifications@github.com
> wrote:

> In external/mrtrix/mrtrix_track.m
> <https://github.com/vistalab/vistasoft/pull/150#discussion_r48129204>:
>
> >  %
> >  % function [status, results, fg, pathstr] = mrtrix_track(files, roi, mask, mode, nSeeds, bkgrnd, verbose)
> >  %
> > -% Provided a csd estimate, generate estimates of the fibers starting in roi
> > +% Provided a csd estimate, generate estimates of the starting in roi
>
> let's say "streamlines/fascicles"?
>
> "streamline" is a default term in mrTrix. In that sense, streamline is
> good.
> "fascicle" is widely used for LiFE.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/vistalab/vistasoft/pull/150/files#r48129204>.
>